### PR TITLE
Add View Helpers section to getting-started

### PR DESCRIPTION
**Pull request by marksiemers**

### CODE REVIEW

File: getting-started/views/view-helpers.md [hunk 1]
```diff
@@ -0,0 +1,198 @@
+# Views Helpers
+
+Views helpers are available from the [Jasper::Helpers](https://github.com/amberframework/jasper-helpers) library.
+
+## Links
+A `link_to` helper is available:
+
+```slim
+== link_to "Home", "/"
```
Comment (drujensen) [inline getting-started/views/view-helpers.md]: Can you change all the `==` to a `=`?  You only use a double `==` when you don't want the html escaped.

**Comment by drujensen** [comment]
Can you merge this with `https://github.com/amberframework/online-docs/blob/master/getting-started/views/jasper-helpers.md`?

This is what is linked in the  TOC @ `https://github.com/amberframework/online-docs/blob/master/SUMMARY.md`

**Comment by marksiemers** [comment]
I can merge, I just have a few questions and things to clarify about merging with the `jasper-helpers.md` file:
- When complete, should `jasper-helpers.md`, `view-helpers.md`, or both be left?
  - If both, which should be linked in the TOC?
- Are these files used as the content for https://amberframework.org/guides as well?
  - That's where I took my cue to name it `view-helpers.md`, that's the [text displayed for the link](https://github.com/amberframework/amberframework.org/blob/master/src/views/home/_sidebar.slang#L96) in the sidebar
- If the naming is up for discussion, I would prefer "View Helpers" over Jasper. That term will be familiar to developers coming from Rails. I know that isn't the main goal, but AFAIK, rails, phoenix, django, nor any of the node frameworks use the term `jasper`. I'm ignorant to its origin though, so maybe there is a strong reason to use it.

**Comment by drujensen** [comment]
All good questions:
- When complete, there should only be one link to helpers left
- This project is used as the guides on the amberframework.org site
- Jasper is a name we gave to the helper sub-project.  We thought that the project could be re-used elsewhere so we wanted to give names to them.
- You may change the filename if you think its better as view-helpers to match the TOC link.

**Comment by marksiemers** [comment]
I kept the `jasper-helpers.md` filename and change the display text in the TOC to say 'View Helpers' - to match the link in the sidebar of the guides.

**Comment by drujensen** [comment]
@marksiemers thank you for the contribution!

**Comment by marksiemers** [comment]
@drujensen - Thanks for accepting the changes. And I'm happy to contribute - Amber and Crystal both look incredibly promising.

**Pull request merged**